pastepwn/database/sqlitedb.py

```diff
@@ -15,10 +15,14 @@ def __init__(self, dbpath="pastepwn"):
         self.logger.debug("Initializing SQLite - {0}".format(dbpath))
 
         # Check if the folder path exists
-        if not os.path.exists(os.path.dirname(dbpath)):
+        if not os.path.exists(dbpath):
             # If not, create the path and the file
-            os.mkdir(os.path.dirname(dbpath))
+            dbdir = os.path.dirname(dbpath)
+            if dbdir != '':
+                os.mkdir(dbdir)
```

Comment:
You are calling mkdir only if `dbdir` is not empty. Shouldn't it be the other way around? Create the directory if `dbdir` is empty? Maybe I am not understanding it fully, because I did not test it locally yet.

Reply:
If `dbdir` is empty `dbpath` is a filename without any folder, i.e. `dbdir` is the current folder and obviously the current folder doesn't need to be created (`os.mkdir('')` actually throws an error).